docs/rst/recording/usage/configuration.rst

```diff
@@ -325,6 +326,18 @@ The recorder output file does support the following configurations:
         - ``string``
         - ``output``
 
+    *   - Timestamp format
```

Comment:
There is no mention to `output` tag in this section so we should add that. We should may include an example of usage.